assets/src/edit-story/components/elementBorder/index.js

```diff
@@ -40,17 +40,24 @@ const borderElementCSS = css`
 const Border = styled.div`
   ${borderElementCSS}
   &:after {
-    content: ' ';
-    ${({ color, left, top, right, bottom, position }) =>
+    ${({ position }) =>
+      !isOutsideBorder({ position }) &&
+      `
+      content: ' ';
+    `}
+    ${({ color, left, top, right, bottom, position, borderRadius }) =>
+      !isOutsideBorder({ position }) &&
       getBorderStyle({
         color,
         left,
         top,
         right,
         bottom,
         position,
+        borderRadius,
       })}
```

Comment:
This would be a bit cleaner as just:

```suggestion
    ${(props) => !isOutsideBorder(props) && getBorderStyle(props)}
```